templates/conll2003/templates.yaml

```diff
@@ -0,0 +1,124 @@
+dataset: conll2003
+templates:
+  3b777e30-bdd1-4c4a-b5d9-3372c7a756f4: !Template
+    id: 3b777e30-bdd1-4c4a-b5d9-3372c7a756f4
+    jinja: "{% set _ner_lable_dict = ({\n0:\"O\",\n1:\"B-PER\",\n2:\"I-PER\",\n3:\"\
+      B-ORG\",\n4:\"I-ORG\",\n5:\"B-LOC\",\n6:\"I-LOC\",\n7:\"B-MISC\",\n8:\"I-MISC\"\
+      \n}) %}\nGenerate named entities from the following sentence. \n{{\"\"}}\n{%\
+      \ for i in tokens -%}\n    {{- \" \" if not loop.last else \"\" -}}\n    {{\
+      \ i }}\n{% endfor %} \n|||\n{% set flag = 0 %}\n{% set outer_cnt = namespace(value=0)\
+      \ -%}\n{% for tok in tokens -%}\n    {% set inner_cnt = namespace(value=0) -%}\n\
```

Comment:
This looks very complex. If there a reason you can't use loop index? 

https://stackoverflow.com/questions/1567291/get-loop-index-of-outer-loop

Reply:
Done.